python/dataproc_templates/pubsublite/README.md

```diff
@@ -0,0 +1,73 @@
+Template for reading files from Cloud Pub/sub Lite and writing them to a BigQuery table.
+
+It uses the [Pubsublite Spark SQL Streaming](https://central.sonatype.com/artifact/com.google.cloud/pubsublite-spark-sql-streaming/1.0.0) to read the streaming data from Pubsublite and [Spark BigQuery connector](https://cloud.google.com/dataproc-serverless/docs/guides/bigquery-connector-spark-example) for writing to BigQuery
+
+## Arguments
+
+* `pubsublite.to.bq.input.subscription`: Cloud Storage location of the input files (format: `gs://bucket/...`)
+* `pubsublite.to.bq.write.mode`: Output write mode (one of: append,overwrite,ignore,errorifexists)(Defaults to append)
+* `pubsublite.to.bq.output.dataset`: BigQuery output dataset name
+* `pubsublite.to.bq.output.table`: BigQuery output table name
+* `pubsublite.to.bq.bucket.name`: Temporary bucket for the Spark BigQuery connector
```

Comment:
If it is temp bucket, please add temp in the templateProperty name

Reply:
Fixed -> [commit](https://github.com/GoogleCloudPlatform/dataproc-templates/pull/658/commits/90477ae3a3561666fa24d39b0427348de969c06c)